index.js

```diff
@@ -0,0 +1,82 @@
+import { DateTime } from 'luxon';
+import { Store } from './storage.js';
+import { UI } from './UI.js';
+import { displayList, displayForm, displayContact } from './display.js';
+
+// Book Class: Represents a Book
+class Book {
+  constructor(title, author, isbn) {
+    this.title = title;
+    this.author = author;
+    this.isbn = isbn;
+  }
+}
+
+// UI Class: Handle UI Tasks
+
+// Store Class: Handles Storage
+
+// Event: Display Books
+document.addEventListener('DOMContentLoaded', UI.displayBooks);
+
+// Event: Add a Book
+document.querySelector('#book-form').addEventListener('submit', (e) => {
+  // Prevent actual submit
+  e.preventDefault();
+
+  // Get form values
+  const title = document.querySelector('#title').value;
+  const author = document.querySelector('#author').value;
+  const isbn = document.querySelector('#isbn').value;
+
+  // Validate
+  if (title === '' || author === '' || isbn === '') {
+    UI.showAlert('Please fill in all fields', 'danger');
+  } else {
+    // Instantiate book
+    const book = new Book(title, author, isbn);
+
+    // Add Book to UI
+    UI.addBookToList(book);
+
+    // Add book to store
+    Store.addBook(book);
+
+    // Show success message
+    UI.showAlert('Book Added', 'success');
+
+    // Clear fields
+    UI.clearFields();
+  }
+});
+
+document.addEventListener('DOMContentLoaded', () => {
+  const currentDate = document.getElementById('current-date');
+  const date = DateTime.now().toLocaleString();
+  if (currentDate) {
+    currentDate.textContent = date;
+  }
+});
+
+document.querySelector('#book-list').addEventListener('click', (e) => {
+  // Remove book from UI
+  UI.deleteBook(e.target);
```

Comment:
-  [ ] I'll suggest you move these event listeners into a function declaration and then call them when needed, :+1: That will make your codebase clean and more easy to maintain. :smile: :wave: